Source/TPCE/Private/Components/ArmComponent.cpp

```diff
@@ -227,35 +227,32 @@ FVector UArmComponent::GetTargetLocation() const
 
 FRotator UArmComponent::GetTargetRotation() const
 {
-	FRotator DesiredRotation;
+	FRotator DesiredRotation = GetComponentRotation();
```

Comment:
With this change DesireRotation may get assigned twice and if GetComponentRotation() has any functional or performance side effects they will be always imposed. If you want to get rid of the goto do 
```
FRotator DesiredRotation;
APawn* OwningPawn;
if (bUsePawnControlRotation && (OwningPawn = Cast<APawn>(GetAttachmentRootActor())))
     DesiredRotation = OwningPawn->GetViewRotation() + GetRelativeRotation();
else 
     DesiredRotation = GetComponentRotation();
```
One may argue that this also changes the code in the sense that now OwningPawn is defined in the method scope and not restricted anymore to the if block but this is just a semantic change that has no impact in the generated instructions.